src/server/graphql/types/CustomPhaseItem.js

```diff
@@ -0,0 +1,38 @@
+import {GraphQLBoolean, GraphQLID, GraphQLInterfaceType, GraphQLNonNull} from 'graphql';
+import {RETRO_PHASE_ITEM} from 'universal/utils/constants';
+import RetroPhaseItem from 'server/graphql/types/RetroPhaseItem';
+import CustomPhaseItemTypeEnum from 'server/graphql/types/CustomPhaseItemTypeEnum';
+import Team from 'server/graphql/types/Team';
+import {resolveTeam} from 'server/graphql/resolvers';
+
+export const customPhaseItemFields = {
```

Comment:
ahhh that's whatcha meant in the figma comment!
I think we can handle this while staying view agnostic.
here's how I imagine it working:
- there is a `startMeeting` mutation
- that mutation grabs each of the phases in the meeting + any team-defined phases, (eg metrics)
- for each phase, it grabs all the active phase items
- it now has a big array of phaseitems [checkin, think-good, think-bad, think-change...]
- the client can decide how to display these. for example it could:
  - show 1 phase item per view
  - show all 3 phase items on the same view, keep them as distinct phase items with separate timers (how retrium works)
  - show all 3 phase items on the same view & when you advance from the first, it auto-advances the remaining phase items in that phase. This can be done with an `autoAdvance` flag that lives on the `CustomPhaseItem`. Another flag might be `viewOnce`, which would live on the FirstCall-main phase item (per our logic, you can only hit the first call view once, after this, it'll push you to the agenda item) 

that way, we'll be able to iterate quickly on the front-end design without touching the backend.